.gitignore

```diff
@@ -3,8 +3,13 @@ __pycache__/
 *.py[cod]
 
 # local Django logs:
-*.log
-
+***.log
```

Comment:
I'm curious, what does "***" do? I haven't seen this gitignore notation before.

Reply:
If you want to ignore files recursively you should use `**` notation.
I wanted to ignore all log file in all project, so I added 2 wild cards to find them in all folders and third wild card as usual to ignore `.log` files.